task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -448,7 +448,7 @@ export default {
       }
     },
     updateTaskStatus(value) {
-      if (value && this.oldTask.status.id!==value.id) {
+      if (value && this.oldTask.status.id !== value.id) {
```

Comment:
```suggestion
      if (value && this.oldTask?.status?.id !== value?.id) {
```
To make sure to not have a nullability problem

Reply:
done !